mintapi/api.py

```diff
@@ -213,7 +214,7 @@ def get_web_driver(email, password, headless=False, mfa_method=None,
     if session_path is not None:
         chrome_options.add_argument("user-data-dir=%s" % session_path)
 
-    driver = Chrome(chrome_options=chrome_options, executable_path="%s" % executable_path)
+    driver = Chrome(options=chrome_options, executable_path="%s" % executable_path)
```

Comment:
Is this just a change in recent Chrome where chrome_options is older / deprecated?